code/game/machinery/bounty_board.dm

```diff
@@ -0,0 +1,215 @@
+GLOBAL_LIST_EMPTY(allbountyboards)
+GLOBAL_LIST_EMPTY(request_list)
+/**
+  * A machine that acts basically like a quest board.
+  * Enables crew to create requests, crew can sign up to perform the request, and the requester can chose who to pay-out.
+  */
+/obj/machinery/bounty_board
+	name = "request kiosk"
+	desc = "Alows you to place requests for goods and services across the station, as well as pay those who actually did it."
+	icon = 'icons/obj/terminals.dmi'
+	icon_state = "request_kiosk"
+	ui_x = 550
+	ui_y = 600
+	light_color = LIGHT_COLOR_GREEN
+	///Static, global list for containing request datums across all request kiosks.
+	var/static/list/request_list
+	///Static, global value so that each request has a universal value for what request number it is.
+	var/static/request_number
+	///Reference to the currently logged in user.
+	var/datum/bank_account/current_user
+	///The station request datum being affected by UI actions.
+	var/datum/station_request/active_request
+
+/obj/machinery/bounty_board/Initialize(mapload, ndir, building)
+	. = ..()
+	GLOB.allbountyboards += src
+	if(building)
+		setDir(ndir)
+		pixel_x = (dir & 3)? 0 : (dir == 4 ? -32 : 32)
+		pixel_y = (dir & 3)? (dir ==1 ? -32 : 32) : 0
+
+/obj/machinery/bounty_board/Destroy()
+	. = ..()
+	GLOB.allbountyboards -= src
+
+/obj/machinery/bounty_board/attackby(obj/item/I, mob/living/user, params)
+	. = ..()
+	if(istype(I,/obj/item/card/id))
+		var/obj/item/card/id/current_card = I
+		if(current_card.registered_account)
+			current_user = current_card.registered_account
+			return TRUE
+		else
+			to_chat(user, "There's no account assigned with this ID.")
+			return TRUE
+	else if(istype(I, /obj/item/bounty_card))
```

Comment:
if you return 100% when you fill the reqs of the first if, you dont need the else.